ui/src/components/homepage/GoodFriday.js

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import {
+    Flex, 
+    Button, 
+    Text,
+    VStack,
+    Stack,
+    Link,
+    SimpleGrid
+  } from '@chakra-ui/react';
+import { AiOutlineDesktop, AiOutlineInstagram } from 'react-icons/ai';
+import { GiPartyPopper } from 'react-icons/gi';
+import { useMediaQuery } from '@chakra-ui/media-query';
+
+const HeroSection = () => {
+  const [isNotSmallerScreen] = useMediaQuery('(min-width:648px)');
+  return (
+    <Flex
+    w={['100%']}
+    h='110vh'
+    // h={isNotSmallerScreen ? '110vh' : '150vh'}
+    bgImage={`url(${'/images/goodfriday/bg.png'})`}
+    bgSize="cover"
+    bgPosition="center center"
+    justify="center">
+      <SimpleGrid spacingY={isNotSmallerScreen ? '25%' : '35%'}>
+        <Text          
+          fontFamily={'NextSoutherlandSerif'}
+          fontSize={isNotSmallerScreen ? '72px' : '50px'}
+          fontWeight={700}
+          lineHeight={['100%','110%']}
+          textColor={'#935963'}
+          textAlign={'center'}
+          marginTop={isNotSmallerScreen ? ['15%','25%'] : '40%'}>
+            GOOD FRIDAY
+            <br/>
+            SERVICE
+        </Text>
+        {/*Session Stream*/}
+        <Stack>
+          <Button
+            //isDisabled='true'
```

Comment:
this should be disabled leading up to GoodFriday. can you do a check date? If it's not April 15th, then disable button. likewise for the Easter homepage